eidas_proxy_service/datamodels.py

```diff
@@ -0,0 +1,129 @@
+"""
+Lightweight data models.
+
+The declaration of a data model is as simple as:
+
+.. code:: python
+
+    class User(DataModel):
+        name = None  # type: str
+        age = None  # type: int
+
+        def validate(self) -> None:
+            ...
+
+Uppercase attributes or attributes starting with an underscore are not considered as fields.
+"""
+import types
+from abc import ABCMeta, abstractmethod
+from collections import OrderedDict
+from typing import Any, Dict, Iterator, List, Mapping, Tuple, Type
+
+from django.core.exceptions import ValidationError
```

Comment:
Not sure whether to reuse Django's ValidationError or create our own.

Reply:
That should be fine.

Reply:
On another thought... This is currently the only thing depending on django, so maybe have our own as an option to have the project independent on django...

Reply:
I'll switch ut to our error during renaming the package.